### DISCUSSION TITLE
v1.3.1 Released

### DISCUSSION THREAD

**Discussion by kasvith** [wavezync/vue3-google-signin#48] [Announcements]
- minor url updates to npm

<hr /><em>This discussion was created from the release <a href='https://github.com/wavezync/vue3-google-signin/releases/tag/v1.3.1'>v1.3.1 Released</a>.</em>

**Comment by oliver-youdooh** [comment]
@kasvith just wanted to say thank you for this awesome library! It works so well!

**Reply by kasvith** [reply]
thank you :)
